### PR TITLE
Forfatterliste

### PR DESCRIPTION
**Pull request by xenma**

### CODE REVIEW

File: app/views/layouts/blacklight.html.erb [hunk 1]
```diff
@@ -0,0 +1,40 @@
+<!DOCTYPE html>
+<html lang="en" class="no-js">
```
Comment (ronan-mch) [inline app/views/layouts/blacklight.html.erb]: should lang be da?

Comment (siglun) [inline app/views/layouts/blacklight.html.erb]: Yeah

File: app/views/catalog/_home_text.html.erb [hunk 2]
```diff
@@ -0,0 +1,9 @@
+<h1 class="col-sm-12 col-sm-offset-4">ADL</h1>
+<div class="searchHome">
+  <div class="col-sm-12">
+    <%= render 'search_form' %>
+  </div>
+  <div class="col-sm-12 col-sm-offset-4">
+    <a href="/?f[cat_ssi][]=person">Forfatteroversigt</a>
```
Comment (ronan-mch) [inline app/views/catalog/_home_text.html.erb]: `&sort=author_ssi+asc`

File: app/views/layouts/blacklight.html.erb (hunk 1 above)
Comment (ronan-mch) [inline app/views/layouts/blacklight.html.erb]: you're on holiday sigge! :sunny:

**Pull request merged**